src/odemis/acq/stream/test/stream_sparc2_hw_sync_test.py

```
@@ -0,0 +1,50 @@
+
+
+# Skip if ubuntu is 20.04 or lower, as nidaqmx does not work there
+# Check using the python version, because that's easier than checking the OS version
+@unittest.skipIf(sys.version_info < (3, 9), "nidaqmx does not work for Ubuntu 20.04 or lower")
+class SPARC2HwSyncTestCase(BaseSPARCTestCase):
+    """
+    Tests to be run with a (simulated) SPARCv2 using a hardware trigger between the
+    e-beam scanner and the CCD/spectrometer.
+    """
+    simulator_config = SPARC2_HWSYNC_CONFIG
+    capabilities = {"ar", "spec", "hwsync", "vector"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)